docs/basics/debugging.md

```diff
@@ -37,6 +34,13 @@ There are three things you have to do for the debug messages to show up on the c
   substrate-contracts-node --dev -lerror,runtime::contracts=debug
   ```
 
+1. __Set the log level of your node to `runtime::contracts::strace` to trace host function calls.
+   These function calls logs will also be displayed in the `Debug message` panel of [contracts-ui](https://contracts-ui.substrate.io).
+   For example, to view these traces in addition to the logs described above:
+  ```
+  substrate-contracts-node --dev -lerror,runtime::contracts=debug,runtime::contracts::strace=trace
```

Comment:
ah I see this is because of https://github.com/paritytech/substrate-contracts-node/pull/56 
Don't we want to maintain it in the docs though since, this is specific to this binary and users could be surprised to not see their logs if they were using another binary (e.g node-template)

Reply:
we might want to make a cleanup pr for this if we do want to remove this everywhere

Reply:
moving that to another PR

Reply:
@HCastano moving `--dev` updates to https://github.com/paritytech/ink-docs/pull/196